main_script.m

```diff
@@ -2,7 +2,9 @@
 %
 %   Each of the scripts below has a bug that needs to be fixed!
 %
-addpath(genpath('..'))
+addpath(strcat(pwd,'\subscripts'))
```

Comment:
I think `pwd` just gets current folder. Make sure it's clear that users need the working directory to be the current folder in matlab. This is the same issue with `genpath`: someone might open matlab before the scripts, so then the current folder is wherever they left off their last matlab session. This `strcat` method is fine.

Reply:
When you run a script matlab asks two things. Either change folder, or add file directory as a path. 
This code works either choice they make.

Reply:
add a comment explaining this.